arch/sim/Kconfig

```diff
@@ -531,16 +531,31 @@ config SIM_QSPIFLASH_PAGESIZE
 		"wrap" causing the initial data sent to be overwritten.
 		This is consistent with standard SPI FLASH operation.
 
+choice
+	prompt "Simulated Prototype of HCI"
+	default SIM_HCISOCKET
+	depends on (HOST_LINUX && SIM_WALLTIME)
```

Comment:
this enables something that should not be enabled by default. this choice should be an option that you can choose to turn on or not. simply add one "none" option and make it default.

Reply:
I will remove the choice option